src/Mocking.jl

```diff
@@ -25,6 +25,23 @@ function activate()
     return nothing
 end
 
+"""
+    Mocking.activate(func)
```

Comment:
FWIW, I would merge this docstring with the one posted above:

```suggestion
    Mocking.activate([func])
```